Doc/library/test.rst

```diff
@@ -445,12 +446,12 @@ The :mod:`test.support` module defines the following functions:
 
 .. function:: match_test(test)
 
-   Match *test* to patterns set in :func:`set_match_tests`.
+   Return whether *test* matches the patterns set in :func:`set_match_tests`.
 
 
-.. function:: set_match_tests(patterns)
+.. function:: set_match_tests(accept_patterns=None, ignore_patterns=None)
 
-   Define match test with regular expression *patterns*.
+   Define match patterns on test filenames and test method names for filtering tests.
```

Comment:
I think this one is a little too terse; something like this? (I've never used this function.)

```suggestion
   Register :mod:`regular expression <re>` patterns for filtering tests. Tests are filtered based on whether filenames or test method names match one or more of the patterns.
```

Reply:
That's still incomplete, as it doesn't explain how accept_patterns and ignore_patterns interact. I haven't used the function either, so I'd prefer to stick with the version Victor signed off on.

Reply:
Sure, let's leave this for now then